mule-extensions-api/src/main/java/org/mule/runtime/extension/api/annotation/ExternalLib.java

```diff
@@ -63,4 +63,15 @@
    * @return The type of library needed
    */
   ExternalLibraryType type();
+
+  /**
+   * @return If provided, suggests a Maven coordinates where the required library can be found. This coordinates should
+   * follow the Maven convention: {@code [groupId]:[artifactId]:[extension]:[classifier]:[version]}
```

Comment:
The standard name is packaging, not extension. We should consider renaming it.

Also, extension and classifier are optional. Are we planning on forcing users to put this values, group artifact and version should be more than enough to specify a jar dependency

Reply:
Yes, you are right.

> extension and classifier are optional
Yes, they are optional, maybe I should improve the Javadoc explaining that.

Reply:
please add a {code} example in the jdoc with the simple case (gav) and other with the full definition